src/tmxrasterizer/tmxrasterizer.cpp

```diff
@@ -101,7 +101,7 @@ int TmxRasterizer::render(const QString &mapFileName,
         break;
     }
 
-    QSize mapSize = renderer->mapSize();
+    QSize mapSize = renderer->mapBoundingRect().size();
```

Comment:
Will need to store the bounding rect and translate the painter based on the `topLeft` of it.